README.md

```diff
@@ -2,3 +2,27 @@
 
 This project contains scripts that are consumed by load-runner, for
  load-testing the [demo cloud app](https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud).
+
+## Setup
+
+There's a setup script at `./setup/usersWorkorders.js` which creates a
+ specified number of users, and creates a workorder for each one. Running it with node on the command line will show details of the parameters that it expects:
+
+``` bash
+node setup/usersWorkorders.js
```

Comment:
I'm not a huge fan of the shebang lines, but it's definitely a possibility! :) I'll try the `npm run setup` way first.